src/helper/virtualizationHelper.js

```diff
@@ -0,0 +1,90 @@
+/**
```

Comment:
Basically this helper contains the removed code in `computeRenderedColumns`, (also the same for `computeRenderedRows`)